tests/macos/xcodeproj/Test-Target-With-Test-Host-Project.xcodeproj/bazelinstallers/install.sh

```diff
@@ -58,55 +62,58 @@ com.apple.product-type.app-extension)
     exit 1
     ;;
 esac
-output="$TARGET_BUILD_DIR/$FULL_PRODUCT_NAME"
 
-mkdir -p $OBJECT_FILE_DIR_normal/$CURRENT_ARCH/
-chmod -R +w $OBJECT_FILE_DIR_normal/$CURRENT_ARCH/
+if [ -n "$input_options" ]; then
```

Comment:
See the comment for the virtual frameworks case, that should probably not be installing or creating anything

Reply:
Ok seeing you answered the other Q here, perhaps then it was an Xcode 14 issue which started creating them on demand. I just wonder if we can simply turn that feature off - they used to have some xcconfigs related to it.

Overall what you've got seems resonable, but if we don't address this issue the issue will live on: but make moving over to our new generator that much better 🥳

Reply:
Yeah tbh I don't know where exactly that happens in Xcode, the behaviour I noticed was that on launch even without triggering a build the fmw files would be created under DD. I mainly want to keep the generator usable while we might to `XCHammer`, also learning how this works is going to help me to better integrate things in `XCHammer`+`xcbuildkit`.